src/manifestoo_core/addon.py

```diff
@@ -53,6 +53,7 @@ def from_addon_dir(
         cls,
         addon_dir: Path,
         allow_not_installable: bool = False,
+        addons_categories: Optional[List[str]] = None,
```

Comment:
I don't think this kind of filtering belongs here, because we can imagine an infinity of different such filters.
Let's keep that in manifestoo for now.